.github/ISSUE_TEMPLATE/Update A-enviar-software-para-revision.md

```diff
@@ -0,0 +1,105 @@
+---
+name: Enviar software para revisión
+about: Usa esta plantilla para enviar un software a revisar en Español (Experimental)
+---
+
+
+Autor Encargado: nombre (@github_handle)  
+Repositorio:   
+Versión Enviada:   
+Editor: TBD  
+Revisor 1: TBD  
+Revisor 2: TBD  
+Archivo: TBD  
+Versión Aceptada: TBD   
+
+---
+
+
+
+-   Pega el archivo DESCRIPTION completo dentro del siguiente bloque de código.
+
+```
+
+```
+
+
+## Alcance 
+
+- Por favor, indica qué categoria(s) aplican a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes (Inglés=](https://ropensci.github.io/dev_guide/policies.html#package-categories). Por favor, tilda todas las apropiadas. Si no estás seguro, te sugerimos que comiences un pre-envío.
+
+	- [ ] recuperación de datos (data retrieval)
+	- [ ] extracción de datos (data extraction)
+	- [ ] munging de datos (data munging)
+	- [ ] disposición o declaración de datos (data deposition)
+	- [ ] automatización de flujos de trabajo (workflow automation)
+	- [ ] control de versiones (version control)
+	- [ ] manejo de citas y bibliometría (citation management and bibliometrics)
+	- [ ] envoltorios de software científico (scientific software wrappers)
+	- [ ] herramientas para trabajo de campo y reproducibilidad (field and lab reproducibility tools)
+	- [ ] ligamientos con software de base de datos (database software bindings)
+	- [ ] datos geoespaciales (geospatial data)
+	- [ ] análisis de texto (text analysis)
+	
+- Explica cómo y por qué el paquete cae dentro de estas categorías (1 a 3 oraciones):
+
+
+- ¿Cuál es la audiencia esperada y las aplicaciones científicas de este paquete? 
+
+- ¿Hay otros paquetes de R que logren el mismo objetivo? Si los hay, ¿cómo se diferencian del tuyo, o alcanzan nuestro [criterio del mejor de su categoría (documento en Inglés)](https://ropensci.github.io/dev_guide/policies.html#overlap)?
+
+-   (Si aplica) ¿Tu paquete cumple con nuestras [guías de _Ética, Privacidad de Datos e Investigación de Sujetos Humanos_ (documento en Inglés)](https://devguide.ropensci.org/policies.html#ethics-data-privacy-and-human-subjects-research)?
+
+- Si ya has hecho una consulta de pre-envío, por favor pega el enlace al issue correspondiente, el post del foro, u otra discusión. Alternativamente, taguea al editor (con @tag) con el que te contactaste.
```

Comment:
speaking of pre-envio that is another translation we will need to create but we can do it at the end since it is basically a shorter version of the submission template.